worlds/inscryption/__init__.py

```diff
@@ -0,0 +1,126 @@
+                new_item = self.create_item(item['name'])
+                self.multiworld.itempool.append(new_item)
+                nb_items_added += 1
+
+        filler_count = len(self.all_locations if self.options.goal != Goal.option_first_act else act1_locations)
+        filler_count -= nb_items_added
+
+        for i in range(filler_count):
+            index = i % len(filler_items)
+            filler_item = filler_items[index]
+            new_item = self.create_item(filler_item['name'])
+            self.multiworld.itempool.append(new_item)
+
+    def create_regions(self) -> None:
+        used_regions = inscryption_regions_all if self.options.goal != Goal.option_first_act \
+            else inscryption_regions_act_1
+        for region_name in used_regions.keys():
+            self.multiworld.regions.append(Region(region_name, self.player, self.multiworld))
+
+        for region_name, region_connections in used_regions.items():
+            region = self.multiworld.get_region(region_name, self.player)
+            region.add_exits(region_connections)
+            region.add_locations({
+                location: self.location_name_to_id[location] for location in regions_to_locations[region_name]
+            })
+
+    def set_rules(self) -> None:
+        Rules.InscryptionRules(self).set_all_rules()
+        if self.options.painting_checks_balancing.value == PaintingChecksBalancing.option_balanced:
+            Rules.InscryptionRules(self).set_painting_rules()
+        elif self.options.painting_checks_balancing.value == PaintingChecksBalancing.option_force_filler and \
+                sum(item.classification == ItemClassification.filler for item in self.multiworld.itempool) >= 2:
+            region = self.multiworld.get_region("Act 1", self.player)
+            loc = next((x for x in region.locations if x.name == "Act 1 - Painting 2"), None)
+            if loc is not None:
+                loc.progress_type = LocationProgressType.EXCLUDED
+            loc = next((x for x in region.locations if x.name == "Act 1 - Painting 3"), None)
+            if loc is not None:
+                loc.progress_type = LocationProgressType.EXCLUDED
+
+    def fill_slot_data(self) -> Dict[str, Any]:
+        return {
+            "death_link": self.options.death_link.value,
+            "act1_death_link_behaviour": self.options.act1_death_link_behaviour.value,
+            "goal": self.options.goal.value,
+            "randomize_codes": self.options.randomize_codes.value,
+            "randomize_deck": self.options.randomize_deck.value,
+            "randomize_sigils": self.options.randomize_sigils.value,
+            "optional_death_card": self.options.optional_death_card.value,
+            "skip_tutorial": self.options.skip_tutorial.value,
+            "skip_epilogue": self.options.skip_epilogue.value,
+            "epitaph_pieces_randomization": self.options.epitaph_pieces_randomization.value
+        }
```

Comment:
You could use this instead (check that the values are of the same format as  before though)
```suggestion
        return self.options.as_dict(
            "death_link",
            "act1_death_link_behaviour",
            "goal",
            "randomize_codes",
            "randomize_deck",
            "randomize_sigils",
            "optional_death_card",
            "skip_tutorial",
            "skip_epilogue",
            "epitaph_pieces_randomization"
        )
```

Reply:
Not all options are to be added in the slot data, so I don't think this can be applied.

Reply:
This would only add the ones I listed

Reply:
Ah, my mistake then.